src/test/groovy/GebishOrgSpec.groovy

```diff
@@ -1,21 +0,0 @@
-import geb.spock.GebSpec
-
-class GebishOrgSpec extends GebSpec {
```

Comment:
This is the only spock test - You need to keep at least one test to verify that the spock integration stll works.